lab-lee/README.md

```diff
@@ -0,0 +1,43 @@
+# Project Overview
+
+The purpose of this project is to create a RESTful API with persistent storage for creating, reading, updating and deleting ducks.
+
+## Duck properties
+  * Ducks have the following properties:
+    * `Name` Every duck has a name.
+    * `Color` Ducks are all a color, and not always yellow.
+    * `Feathers` Refers to the # of feathers, represented as a string.
+    * `Id` Every duck is given a random id number that also corresponds to its filename in the persistent database.
+
+## How to Install the Project
+
+Check out the [package.json] (https://github.com/broxsonl/lab-06-tcp-chat/blob/master/lab-lee/package.json) file for the project to see the specific Dependencies required to run this project or develop for it.
+
+## How to Start the Server
+
+To start the server, go to the root directory of the project and run the following command:
+
+        $ node server.js
+
+That should start the server. If you're unsure, you should receive a console log to your CLI that lets you know the server has started running at either a port you specified or the default port 3000.
+
+## This Duck API's Routes
+
+The following GET and DELETE paths are valid routes that may be queried with requests. Keep in mind, [PORT] represents where the port number should be passed in.:
+
+        * `$ localhost:[PORT]/hello ` Returns a 'Hey There' from the server.
+        * `$ localhost:[PORT]/api/duck?id=[Id]` Returns a duck if an existing duck Id is entered in place of [Id].
```

Comment:
Because you refactored to use url params, you'll need to change your README to reflect this. The route would now be .../api/duck/id